### DISCUSSION TITLE
Welcome to Mochi MQTT Organisation

### DISCUSSION THREAD

**Discussion by mochi-co** [mochi-mqtt/server#271] [Announcements]
As of today, the ownership of mochi-co/mqtt has been transferred to a new Mochi MQTT organisation! 

**A little history about this project**
This project started in rather humble origins in late 2017, when I was quite interested in developing a cloud-free home automation system, and needed a message transfer system to move telemetric and event data around between devices. After considerable research, I settled on MQTT as a suitable internal protocol. At the time, there were only limited options for an embeddable MQTT server written in Go, and so I embarked on creating my own. Two things happened during this time - first, Home Assistant and HomeKit both made large strides in their usability and cloud-free functionality. Second, I became entirely interested in MQTT in-and-of itself, abandoned the home-automation project, and jumped headfirst into everything MQTT. 

After various experiments and prototypes, I began developing an initial alpha version supporting limited MQTTv3 in [September 2019](https://github.com/mochi-mqtt/server/commit/0c7e59cc85a4b5e0a5fab2b8a871c249df3c14cb). A limited public release was made in [February 2020](https://github.com/mochi-mqtt/server/commit/073d825632257a8d886cc54c6c968929c1e54cfe) under the MIT license in case anybody else might find the code useful. I also continued to use it to power my own side projects and commercial endeavours.

The project has been refactored and rewritten, driven in part by my own sense of perfectionism and vague ambition to create the best embeddable MQTT broker available for Go. Over this time, we've picked up stars and wonderful contributors, many suggestions and issues, and evolved into a mature, stable MQTT solution being used in many production environments. I am proud to say that the Mochi-MQTT we see today is the product of a lot of hard and passionate work from many contributors around the world who share this same goal.

While it was is feasible to keep the repo on my (mochi-co) own personal account indefinitely, I believe the project has great potential which would be faster unlocked when a sense of ownership is shared by those who use it most. For this reason, we are becoming an organisation.   

**The Future of Mochi MQTT**
The path to becoming a well-functioning organisation is resolved with a single step, and there will be many things to do, including the selection of collaborators, organisation policies and codes, the creation of teams, etc. We have an excellent foundation to work with.

However, the first major changes will include:
* Open discussion about roadmap items and future planning.
* Creation of new working groups to implement better integration with opentelemetry and prometheus, as well as a formal discovering around robust, cloud-native clustering support, as these are the two presently most requested improvements.
* Changes to branch and pull request approval requirements to allow for better PR democratisation and safeguarding.
* Discussion to potentially replace PAHO interoperability testing with a new modern equivalent which fully supports all MQTTv5 features.

All of this and more, over the coming weeks.

**FAQ**
_Will the licensing change?_ 
No. Mochi-MQTT will remain MIT licensed, as this is best for the organisations and contributors who use the software.

_Will you (mochi-co) be less involved?_ 
No. I plan to remain at the head of the project and help guide the development plans and future goals, review code, contribute to features, and fix bugs. This change is intended to facilitate increased collaboration and project maturation.

Questions and comments are welcome and encouraged.

**Comment by vish74** [comment]
Thank you for the announcement and sharing the history! Great work ! What would be the best way to contact you? Can you please share email?

**Reply by mochi-co** [reply]
No email is currently available, but you can contact me here. A private discussion can be arranged if the matter is confidential in nature 👍🏻

**Comment by dgduncan** [comment]
Just wanted to say I also love the logo! Makes me think of a pink/salmon colored mochi haha.

**Reply by mochi-co** [reply]
Only just seen this, thank you 🙂

**Comment by tobyzxj** [comment]
Great work， I will continue to follow the development of this project

**Reply by mochi-co** [reply]
Thank you!

**Comment by lanfengye2008** [comment]
This project is very good. Do you have any relevant routes for the cluster?

**Reply by bkupidura** [reply]
Not a author of `mochi-mqtt`, but feel free to check https://github.com/bkupidura/broker-ha which is `mochi-mqtt` based HA ready mqtt broker.
